### DISCUSSION TITLE
Growth Estimates

### DISCUSSION THREAD

**Discussion by gabrielomana** [dpguthrie/yahooquery#232] [Q&A] [answered]
Would it be possible to obtain the data from the "Growth Estimates" section located in the analysis section of Yahoo Finance? Specifically, I am in need of "Growth Estimates Next 5 Years" and "Growth Estimates Past 5 Years".

**Comment by ms82494** [comment, marked as answer]
> Would it be possible to obtain the data from the "Growth Estimates" section located in the analysis section of Yahoo Finance? Specifically, I am in need of "Growth Estimates Next 5 Years"
```
In [18]: yq.Ticker('AAPL').earnings_trend['AAPL']['trend'][-2]['growth']
Out[18]: 0.0698
```
> and "Growth Estimates Past 5 Years".

```
In [17]: yq.Ticker('AAPL').earnings_trend['AAPL']['trend'][-1]['growth']
Out[17]: 0.22156
```

You'll probably want to access this in a more robust way (without relying on the internal order of the dictionaries in the list), but this should get you going.
